en-US/mozorg/internet-health/privacy-security.lang

```diff
@@ -0,0 +1,188 @@
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/privacy-security.html
```

Comment:
It should be https://www-dev.allizom.org/it/internet-health/privacy-security/ (also fix following line)